src/pages/patientView/PatientViewPage.tsx

```diff
@@ -254,6 +256,18 @@ export default class PatientViewPage extends React.Component<IPatientViewPagePro
                                             patientViewPageStore.sampleToDiscreteGenePanelId.result,
                                             patientViewPageStore.genePanelIdToEntrezGeneIds.result);
     }
+    
+    @autobind
+    @action toggleGenePanelModal(genePanelId?:string|undefined) {
+        this.genePanelModal = {
+            isOpen: !this.genePanelModal.isOpen,
+            genePanelId: typeof genePanelId === 'string' ? genePanelId : ''
```

Comment:
`genePanelId: genePanelId || ''` is easier to read

Reply:
I agree, and have updated according to your suggestion. However, this function was passed directly to handle the `onHide` event of Bootstrap Modal and `onClick` event of Bootstrap Button, and whenever the function was called, an event object would get passed to the parent component and be seen as `genePanelId`. In order to avoid this, I modified the shared `GenePanelModal` component like below:

```
    @autobind handleClose() {
        this.props.onHide();
    }
    
    render() {
        return (
            <Modal
                show={this.props.show}
                onHide={this.handleClose} // instead of onHide={this.props.onHide}
                ...
```